packages/devtools_app/lib/src/memory/memory_allocation_table_view.dart

```diff
@@ -58,28 +64,166 @@ class AllocationTableViewState extends State<AllocationTableView>
         controller.computeAllLibraries(rebuild: true);
       });
     });
+
+    addAutoDisposeListener(controller.updateClassStackTraces, () {
+      setState(() {});
+    });
+
+    addAutoDisposeListener(controller.treeChangedNotifier, () {
+      if (controller.isTreeChanged) {
+        setState(() {});
+      }
+    });
+  }
+
+  /// Construct all objects tracked and their stacktraces.
+  List<Widget> allTrackedCreations(ClassRef classRef) {
```

Comment:
This should use ListView or preferably our TreeTable class instead. As written this will kill Flutter Web performance once there are a lot of allocations.
If we need to add some features to the TreeTable I think that would be world be worth it. For example, having an option to truncate to only show the first N (4) children by default and support expanding to show the rest of the content.

Reply:
Yeah I've re-written using TreeTable can't nest treeTable in another treetable but I'll look at how to change Table to support that.